dev/com.ibm.ws.ejbcontainer.timer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -36,3 +36,9 @@ persistentExecutorRef.desc=Schedules and runs EJB persistent timer tasks.
 nonPersistentContextServiceRef=Non-persistent Timer Thread Context Propagation
 nonPersistentContextServiceRef$Ref=Thread context propagation reference
 nonPersistentContextServiceRef.desc=The context service is used to manage context propagation to non-persistent timer method threads.
+
+missedPersistentTimerAction=Missed persistent timer action
+missedPersistentTimerAction.desc=Specifies the action to perform when the expiration of an interval or schedule based persistent timer is missed. One or more expirations of a persistent timer are classified as missed if the current expiration is scheduled prior to application server start or the next expiration is scheduled prior to the current time. The default action when failover of persistent timers is enabled is ONCE, otherwise the default action is ALL.
+missedPersistentTimerAction.ALL=The timeout method is invoked immediately for all missed expirations. When multiple expirations have been missed for the same timer, each invocation will occur synchronously until all missed expirations have been processed, then the timer will resume with the next future expiration. 
+missedPersistentTimerAction.ONCE=The timeout method is invoked once immediately. All other missed expirations are skipped and the timer will resume with the next future expiration. 
+missedPersistentTimerAction.NONE=No action is taken. All missed expirations that occurred prior to application server start are skipped and the timer will resume with the next future expiration. Missed expirations that occur after application server start are handled with the ONCE missed timer action.   
```

Comment:
Change "occurred prior to application server start" to "occurred before application server start".
Change "the timer will resume " to "the timer resumes".

Reply:
done